### PR TITLE
Events for 2022-11-30

### PR DESCRIPTION
**Pull request by andrewpollack**

### CODE REVIEW

**Comment by mariannegoldin** [comment]
Small nit: the below Munich event on 11/30 could be re-arranged so that it appears after the Milan event. 
```
* 2022-11-30 | Munich, DE + Virtual | [Rust Munich](https://www.meetup.com/rust-munich/)
* [**Rust Munich 2022 / 3 - hybrid**](https://www.meetup.com/rust-munich/events/289065390/)
```

File: draft/2022-11-30-this-week-in-rust.md [hunk 1]
```diff
@@ -109,6 +103,8 @@ Rusty Events between 2022-11-30 - 2022-12-28 🦀
     * [**Rust Hack and Learn**](https://berline.rs/2022/12/06/rust-hack-and-learn.html)
 * 2022-12-06 | Virtual (Buffalo, NY, US) | [Buffalo Rust Meetup](https://www.meetup.com/buffalo-rust-meetup/)
     * [**Buffalo Rust User Group, First Tuesdays**](https://www.meetup.com/buffalo-rust-meetup/events/hlgvxsydcqbjb/)
+* 2022-12-07 | Virtual (Dallas, TX, US) | [Dallas Rust](https://www.meetup.com/Dallas-Rust/)
```
Comment (mariannegoldin) [inline draft/2022-11-30-this-week-in-rust.md:106]: It looks like this got moved to 12/13:
https://www.meetup.com/dallas-rust/events/vndgwsydcqbrb/

**Review by mariannegoldin — APPROVED** [review]
Approved with one correction. Thank you!

File: draft/2022-11-30-this-week-in-rust.md (hunk 1 above)
Comment (andrewpollack) [reply draft/2022-11-30-this-week-in-rust.md]: Good eye -- moved

**Comment by andrewpollack** [comment]
Thank you! Implemented both adjustments

**Pull request merged**